plugins/Live/stylesheets/live.less

```diff
@@ -69,35 +70,39 @@
 }
 
 #visitsLive .visits-live-launch-visitor-profile {
-  color: @theme-color-text;
-  line-height: 200%;
+    color: @theme-color-text;
+    line-height: 200%;
 
-  span {
-    display: block;
-    vertical-align: middle;
+    span {
+        display: block;
+        vertical-align: middle;
 
-    &.icon-visitor-profile {
-      display: inline;
-      font-size: 16px;
-      line-height: 16px;
-      vertical-align: middle;
+        &.icon-visitor-profile {
+            display: inline;
+            font-size: 16px;
+            line-height: 16px;
+            vertical-align: middle;
+        }
     }
-  }
 
-  .icon-visitor-profile {
-    font-size: 120%;
-  }
+    .icon-visitor-profile {
+        font-size: 120%;
+    }
 }
 
 .theWidgetContent .card .card-content #visitsTotal {
-  margin-left: -20px;
-  width: ~"calc(100% + 40px)";
+    margin-left: -20px;
+    width: ~"calc(100% + 40px)";
+}
+
+#visitsTotal {
+    overflow-x: scroll;
```

Comment:
This will actually always add a scrollbar to the widget. Would be good to only have it, if it is needed.